pycbc/io/live.py

```diff
@@ -266,36 +271,77 @@ def __init__(self, ifos, coinc_results, **kwargs):
             psds_lal[ifo] = fseries
         make_psd_xmldoc(psds_lal, outdoc)
 
-        # source probabilities estimation
+        # P astro calculation
+        if 'padata' in kwargs:
+            padata = kwargs['padata']
+            trigger_data = {
+                'mass1': sngl_populated.mass1,
+                'mass2': sngl_populated.mass2,
+                'spin1z': sngl_populated.spin1z,
+                'spin2z': sngl_populated.spin2z,
+                'network_snr': network_snrsq ** 0.5,
+                'far': far}
+            horizons = {ifo: self.psds[ifo].dist}
```

Comment:
`for ifo in …` missing?

Reply:
interesting, I wonder what this did given that it didn't produce an error

Reply:
It probably created a one-item dict with a random ifo from the code above.